README.md

```diff
@@ -7,14 +7,12 @@ Hazel is primarily an early-stage interactive application and rendering engine f
 ## Getting Started
 Visual Studio 2017 or 2019 is recommended, Hazel is officially untested on other development environments whilst we focus on a Windows build.
 
-You can clone the repository to a local destination using git:
+Start by cloning the repository with `git clone --recursive https://github.com/TheCherno/Hazel`.
 
-`git clone --recursive https://github.com/TheCherno/Hazel`
-
-Make sure that you do a `--recursive` clone to fetch all of the submodules!
+If the repository was cloned non-recursively previously, use `git submodule update --init` to clone the necessary submodules.
 
 ## The Plan
-The plan for Hazel is two-fold: to create a powerful 3D engine, but also to serve as an education tool for teaching game engine design and architecture. Because of this the development inside this repository is rather slow, since everything has to be taught and implemented on-camera. There is a much more advanced version of the engine in a private repository called `Hazel-dev`, accessible to supporters on [Patreon](https://patreon.com/thecherno). The plan for this project is to mostly take already implemented code from the `Hazel-dev` repository and integrate it into this one, done within videos and supported by explanations.
+The plan for Hazel is two-fold: to create a powerful 3D engine, but also to serve as an education tool for teaching game engine design and architecture. Because of this the development inside this repository is rather slow, since everything has to be taught and implemented on-camera. There is a much more advanced version of the engine in a private repository called `Hazel-dev`, accessible to supporters on [![Patreon](https://img.shields.io/badge/%40thecherno--green.svg?style=social&logo=Patreon)](https://patreon.com/thecherno). The plan for this project is to mostly take already implemented code from the `Hazel-dev` repository and integrate it into this one, done within videos and supported by explanations.
```

Comment:
I would not add the patreon shield in the middle of the text, maybe just link it there, and add the shields at the bottom, like the rest of them